agenta-backend/agenta_backend/services/selectors.py

```diff
@@ -9,22 +9,32 @@ async def get_user_and_org_id(session) -> Dict[str, str]:
         session (SessionContainer):  Used to store and manage the user's session data
 
     Returns:
-        A dictionary containing the user_id and organization_id.
+        A dictionary containing the user_id and a list of the user's organization_ids.
     """
-    return {"uid": "0", "organization_id": "0"}
+    user_uid_id = "0"
+    user_id, org_ids = await get_user_objectid(user_uid_id)
```

Comment:
this is a very weird function. 
get_user_objectid in my understanding should return a user_objectid, not a tuple ! This is confusing

Reply:
get_user_objectid is only used in the function above it (get_user_and_org_id), in ee it is unchanged.